source/_docs/installation/virtualenv.markdown

```diff
@@ -1,20 +1,27 @@
 ---
 layout: page
-title: "Installation in virtualenv"
-description: "Instructions how to install Home Assistant in a virtual environment."
+title: "Installation: Python virtual envenvironment"
```

Comment:
What's the advantage of starting to prefix the title with the section where the doc is located?

Also, there is a typo in `envenvironment`.